### PR TITLE
[Spanish] localize 'Site Reliability Engineering' 

### LINKED ISSUE #1774: [Spanish] localize 'Site Reliability Engineering'
Localize the `content/en/site-reliability-engineering.md` document to Spanish

### PR DESCRIPTION
**Pull request by Azanul**
### Describe your changes
Translated site-reliability-engineering.md in spanish

### Related issue number or link (ex: `resolves #issue-number`)
resolves #1774 

### Checklist before opening this PR (put `x` in the checkboxes)
- [X] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [X] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/es/site-reliability-engineering.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+category: Concepto
+tags: ["methodology", "", ""]
+---
+
+## ¿Qué es?
+
+Site Reliability Engineering o SRE es una disciplina que combina operaciones e ingeniería de software.
+Este último se aplica a problemas de infraestructura y operaciones, específicamente.
+Es decir, en lugar de crear características del producto, los ingenieros de confiabilidad del sitio crean sistemas para ejecutar aplicaciones.
+Hay similitudes con [DevOps](/devops/), pero mientras DevOps se enfoca en llevar el código a producción,
```
Comment (ramrodo) [inline content/es/site-reliability-engineering.md:13]:
```suggestion
Hay similitudes con [DevOps](/es/devops/), pero mientras DevOps se enfoca en llevar el código a producción,
```

File: content/es/site-reliability-engineering.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+SRE garantiza que el código que se ejecuta en producción funcione correctamente.
+
+## Problema que aborda
+
+Asegurar que las aplicaciones se ejecuten [confiablemente](/reliability/) requiere múltiples capacidades,
+desde la supervisión del rendimiento, las alertas, la [depuración](/debugging/) hasta la resolución de problemas.
```
Comment (ramrodo) [inline content/es/site-reliability-engineering.md:19]:
```suggestion
desde la supervisión del rendimiento, las alertas, la [depuración](/es/debugging/) hasta la resolución de problemas.
```

File: content/es/site-reliability-engineering.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+Sin estos, los operadores del sistema solo pueden reaccionar a los problemas en lugar de trabajar de manera proactiva para evitarlos.
+— el tiempo de inactividad solo se convierte en una cuestión de tiempo.
+
+## ¿Cómo ayuda?
+
+Un enfoque SRE minimiza el costo, el tiempo y el esfuerzo del proceso de desarrollo de software
+mejorando continuamente el sistema subyacente.
+El sistema mide y monitorea continuamente la infraestructura y los componentes de la aplicación.
+Cuando algo sale mal, el sistema indica a los ingenieros de confiabilidad del sitio cuándo, dónde y cómo solucionarlo.
+Este enfoque ayuda a crear sistemas de software altamente [escalables](/scalability/) y confiables mediante la automatización de tareas operativas.
```
Comment (ramrodo) [inline content/es/site-reliability-engineering.md:29]:
```suggestion
Este enfoque ayuda a crear sistemas de software altamente [escalables](/es/scalability/) y confiables mediante la automatización de tareas operativas.
```

**Review by ramrodo — CHANGES_REQUESTED** [review]
Thanks for your contribution!

Just some suggestions to update the concepts already localized adding `/es/`

**Comment by Azanul** [comment]
> Thanks for your contribution!
> 
> Just some suggestions to update the concepts already localized adding `/es/`

Applied the suggested changes

**Comment by electrocucaracha** [comment]
@Azanul Spanish translations must target to `dev-es` branch. Maybe you'll require to redo this PR using `dev-es` branch as a base instead of `main`

**Review by ramrodo — CHANGES_REQUESTED** [review]
@Azanul could you keep only the changes related to the localization and remove the GitHub Actions upgrades and SCSS, please?

**Review by krol3 — CHANGES_REQUESTED** [review]

**Comment by Azanul** [comment]
For now, I've force pushed to resolve it.

**Comment by ramrodo** [comment]
> @ramrodo @krol3 There's something wrong here. I didn't change any of those. Someone else has also faced the same issue. #1855

This error is probably due to a rebase problem. We should rebase our branch and keep it updated against the `main` branch. With that, the PR/branch should not include other changes.

**Review by ramrodo — CHANGES_REQUESTED** [review]
Could you do a rebase against the upstream **dev-es** branch, please? It seems there are some conflicts with the `content/es/.wordlist.txt` file.

**Comment by Azanul** [comment]
@ramrodo Resolved conflicts

File: content/es/site-reliability-engineering.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+Este último se aplica a problemas de infraestructura y operaciones, específicamente.
+Es decir, en lugar de crear características del producto, los ingenieros de confiabilidad del sitio crean sistemas para ejecutar aplicaciones.
+Hay similitudes con [DevOps](/es/devops/), pero mientras DevOps se enfoca en llevar el código a producción,
+SRE garantiza que el código que se ejecuta en producción funcione correctamente.
+
+## Problema que aborda
+
+Asegurar que las aplicaciones se ejecuten [confiablemente](/es/reliability/) requiere múltiples capacidades,
+desde la supervisión del rendimiento, las alertas, la [depuración](/es/debugging/) hasta la resolución de problemas.
+Sin estos, los operadores del sistema solo pueden reaccionar a los problemas en lugar de trabajar de manera proactiva para evitarlos.
```
Comment (ramrodo) [inline content/es/site-reliability-engineering.md:20]: Remove the final dot. This dot is not in the original English file and this was added automatically by using a machine translator:

```suggestion
Sin estos, los operadores del sistema solo pueden reaccionar a los problemas en lugar de trabajar de manera proactiva para evitarlos
```

**Review by ramrodo — CHANGES_REQUESTED** [review]
Suggestion made

**Review by ramrodo — APPROVED** [review]
Thanks!
/lgtm

**Review by krol3 — APPROVED** [review]
ltgm

**Pull request merged**